src/util/tailchart.js

```diff
@@ -80,7 +81,7 @@ export default class TailChart {
 
   updateChartLabels(mean) {
     let word = this.whatAreWeRecording;
-    if (this.tailDirection === "null") {
+    if (this.tailDirection == null || this.tailDirection == 'null') {
```

Comment:
@aryue123321 you might need to do this on your tail charts

Reply:
it works